README.md

```diff
@@ -11,7 +11,7 @@ Sterling, and Birkedal. Code has been incorporated from [redtt](https://www.gith
 
 ## building
 
-cooltt has been built with OCaml 4.09.0 with [opam 2.0.5](https://opam.ocaml.org/). Once
+cooltt has been built with OCaml 4.10.0 with [opam 2.0.5](https://opam.ocaml.org/). Once
```

Comment:
i'm at least on opam 2.0.6; i don't know what other folks are using.

Reply:
It's fixed in the CI script.
https://github.com/avsm/setup-ocaml/blob/0662d851bd96d829f21062d9c9c7ab07ac65ebc2/src/installer.ts#L51

Reply:
ah, gotcha!